### ISSUE TITLE
Add term spastic dystonia

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10338]
**Preferred term label:**
Spastic dystonia

**Synonyms**
-
**Definition (free text, please give PubMed ID)**
The term spastic dystonia refers to involuntary, spontaneous tonic muscle contraction in the context of an upper motor neuron (UMN) lesion.
PMID: 36940234

**Parent term (use hpo.jax.org/app)**
Dystonia HP:0001332
--> Spasticity and dystonia are phenomenologically and pathophysiologically distinct entities. Spastic dystonia represents a subset of dystonia, but with pathophysiological mechanisms more in common with spasticity. (PMID 36940234)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Part of Upper Motor Neuron Syndrome. For example in: Multiple Sclerosis (OMIM126200)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
this is a bundled term and it would be better to code both Dystonia and Spasticity separately!
